### PR TITLE
Add dup expr support

### PR DESCRIPTION
**Pull request by alexispires**
This PR adds support for ``dup`` expression

### CODE REVIEW

**Comment by alexispires** [comment]
Relate to #80

**Comment by alexispires** [comment]
@googlebot I signed it!

**Comment by alexispires** [comment]
@googlebot I fixed it.

File: expr/dup.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	RegAddr uint32
+	RegDev  uint32
+}
+
+func (d *Dup) marshal() ([]byte, error) {
+	attrs := []netlink.Attribute{
+		{Type: unix.NFTA_DUP_SREG_ADDR, Data: binaryutil.BigEndian.PutUint32(d.RegAddr)},
+	}
+
+	if d.RegDev != 0 {
```
Comment (sbezverk) [inline expr/dup.go]: In some other expressions second register may carry value of 0.  If there is such case for Dup expression,  then this check will break the expression. How about to use a boolean flag indicating whether RegDev is set or not?

**Comment by stapelberg** [comment]
Thanks!

**Pull request merged**